README.md

```diff
@@ -29,8 +30,9 @@ import ReactDOM from 'react-dom';
 import 'froala-editor/js/froala_editor.pkgd.min.js';
 
 // Require Editor CSS files.
+import 'froala-editor/js/froala_editor.min.js';
```

Comment:
This shouldn't be included as it is already included by the `froala_editor.pkgd.min.js` file. Also, we shall include the `froala_editor.pkgd.min.css` instead of `froala_editor.min.css`.

Reply:
Perhaps I should create an issue? The CSS did not show up properly for the table until I added this file.

Reply:
It was because of the CSS `froala_style.min.css`, not the JS.